foundationdb/src/future.rs

```diff
@@ -48,6 +48,8 @@ impl Drop for FdbFuture {
     }
 }
 
+unsafe impl Send for FdbFuture {}
```

Comment:
I worry about marking FdbFuture Send. I can imagine a future where we re-implement the the C++ client code in native Rust and directly on Tokio, etc. In that context, Future may not be send, right?

That is, my concern is that we may paint ourselves into a corner by making FdbFuture Send now. So I might suggest holding off on making this Send (unless there is a need to do this now).

Reply:
I agree. I'll add a patch to fix it.